controllers/registry.go

```diff
@@ -502,7 +519,7 @@ func (r *VeleroReconciler) getSecretNameAndKey(credential *corev1.SecretKeySelec
 	return secretName, secretKey
 }
 
-func (r *VeleroReconciler) parseAWSSecret(secret corev1.Secret, secretKey string) (string, string, error) {
+func (r *VeleroReconciler) parseAWSSecret(secret corev1.Secret, secretKey string, awsProfile string) (string, string, error) {
 
 	AWSAccessKey, AWSSecretKey := "", ""
 	// this logic only supports single profile presence in the aws credentials file
```

Comment:
Please remove this comment, as this PR adds support for multiple profiles.

Reply:
Removed.